setup.json

```diff
@@ -7,18 +7,17 @@
     "license": "MIT",
     "classifiers": [
         "Environment :: Plugins",
-        "Programming Language :: Python :: 3.6",
-        "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
```

Comment:
Any reason not to support 3.10?

Reply:
Not really. I did not test it and still not using 3.10 in my local environments. Possibly in near future.